### PR TITLE
Updated Bug report template  

### PR DESCRIPTION
**Pull request by Chandra-Sekhar-Bala**
As we discussed @designe , update bug-report template

### CODE REVIEW

**Comment by Chandra-Sekhar-Bala** [comment]
Check this one please @yeonns  @designe

**Review by yeonns — APPROVED** [review]
Thank you for applying new bug report template 👍

**Comment by Chandra-Sekhar-Bala** [comment]
@yeonns @designe  Wouldn't it be better to have a communication channel and a Twitter handle for this organization?

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: 🐞 Bug
```
Comment (designe) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:1]: Cute 🐞 lady bug kk

In this repository, we already have a mark-down template.
So, It seems good to keep the existing name as it is.
🐞 Bug -> 🐞 Bug report

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+description: File a bug/issue
```
Comment (designe) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:2]: Here is also good to keep the existing name.
File a bug/issue -> Create a report to help us improve

**Review by designe — COMMENTED** [review]
Thanks for your contribution ;)
This is a PR that has already been verified once in https://github.com/euphony-io/euphony-collector/pull/18.

So, please check my 2 comments.
After fixed it, let me approve :)

**Comment by designe** [comment]
> @yeonns @designe Wouldn't it be better to have a communication channel and a Twitter handle for this organization?

Now, we have a github discussion tab :)
I think it's enough to use discussions as the our main communication port.
https://github.com/orgs/euphony-io/discussions

if we need to make multi communication port, let me consider!
thanks for your suggestion :)

**Comment by Chandra-Sekhar-Bala** [comment]
> > @yeonns @designe Wouldn't it be better to have a communication channel and a Twitter handle for this organization?
> 
> We have a github discussion tab currently :) I think it's enough to use discussions as the our main communication port. https://github.com/orgs/euphony-io/discussions
> 
> if we need to make multi communication port, let me consider! thanks for your suggestion :)

**Review by designe — APPROVED** [review]
Thanks for applying the review 👍 
We will have more convinient issue report through your contribution  ;)
https://promise.is/verified/euphony/certificates/Chandra-Sekhar-Bala

**Pull request merged**